webapp/src/app/shared/settings.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class SettingsService {
+	private static readonly wrapSettingName = 'wrapEventEditorLines';
+	
+	private static loadBooleanOrDefault(key: string, defaultValue: boolean): boolean {
+		const loadedValue = localStorage.getItem(SettingsService.wrapSettingName);
+		return loadedValue === null? defaultValue : (loadedValue === 'true');
+	}
+	
+	get wrapEventEditorLines() {
+		return SettingsService.loadBooleanOrDefault(SettingsService.wrapSettingName, true);
+	}
+	set wrapEventEditorLines(value: boolean) {
```

Comment:
I would personally make all settings into observables, so they can be changed on the fly and can be safely used inside html templates without functions getting called all the time.

So basically something like this: 
```typescript
@Injectable()
export class SettingsService {
	private static readonly wrapSettingName = 'wrapEventEditorLines';
	
	private wrapEditorLines = new BehaviorSubject(
		SettingsService.loadBooleanOrDefault(SettingsService.wrapSettingName, true)
	);
	
	private static loadBooleanOrDefault(key: string, defaultValue: boolean): boolean {
		const loadedValue = localStorage.getItem(SettingsService.wrapSettingName);
		return loadedValue === null ? defaultValue : (loadedValue === 'true');
	}
	
	getWrapEventEditorLines() {
		return this.wrapEditorLines.asObservable();
	}
	
	setWrapEventEditorLines(val: boolean) {
		localStorage.setItem(SettingsService.wrapSettingName, val.toString());
		this.wrapEditorLines.next(val);
	}
}
```

Now in the event editor this can be used like that:

```typescript
export class EventEditorComponent implements OnChanges, OnInit {
	[...]
	$wrapText: Observable<boolean>;

	constructor(
		private helper: EventHelperService,
		private addEvent: AddEventService,
		settingsService: SettingsService
	) {
		this.$wrapText = settingsService.getWrapEventEditorLines();
	}
	[...]
```

```html
<div class="event-container" [ngClass]="{'wrapping': $wrapText | async}" fxLayout="row">
	[...]
```

The benefit of this is that the option can be changed at any time and is immediately reflected in the ui without reloading the component or calling a get function over and over again.

That said it's good how it is. There is no way to change the option without reloading the component anyway. Just for future reference.